src/schema/formats.js

```diff
@@ -8,6 +8,12 @@ export function isValidVersionString(version) {
   if (typeof version !== 'string') {
     return false;
   }
+  // Firefox's version format is laxer than Chrome's, it accepts:
+  // https://developer.mozilla.org/en-US/docs/Toolkit_version_format
+  // We choose a slightly restricted version of that format (but still more
+  // permissive than Chrome) to allow Beta addons, per:
+  // https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Maintenance
+  version = version.replace(/(a|alpha|b|beta|pre|rc)\d*$/, '');
```

Comment:
If we're more permissive than Chrome, we should add a warning/notice about using version strings with a "toolkit" format; that way WebExtension developers won't think their add-on will work unmodified in Chrome (cross-browser help is a goal of the linter).